docs/designers-developers/developers/block-api/block-edit-save.md

```diff
@@ -9,85 +9,130 @@ The `edit` function describes the structure of your block in the context of the
 {% codetabs %}
 {% ESNext %}
 ```jsx
-edit: () => {
-	return <div>Your block.</div>;
-}
+import { useBlockProps } from '@wordpress/block-editor';
+
+// ...
+const blockSettings = {
+	apiVersion: 2,
+	
+	// ... 
+
+	edit: () => {
+		const blockProps = useBlockProps();
+
+		return <div {...blockProps}>Your block.</div>;
+	}
+};
 ```
 {% ES5 %}
 ```js
-// A static div
-edit: function() {
-	return wp.element.createElement(
-		'div',
-		null,
-		'Your block.'
-	);
-}
+var blockSettings = {
+	apiVersion: 2,
+
+	// ...
+
+	edit: function() {
+		var blockProps = wp.blockEditor.useBlockProps();
+
+		return wp.element.createElement(
+			'div',
+			blockProps,
+			'Your block.'
+		);
+	}
+};
 ```
 {% end %}
 
-The function receives the following properties through an object argument:
+### block wrapper props
 
-### attributes
+The first thing to notice here is the use of the `useBlockProps` React hook on the block wrapper element. In the example above, the block wrapper renders a "div" in the editor, but in order for the Gutenberg editor to know how to manipulate the block, add any extra classNames that are needed for the block... the block wrapper element should apply props retrieved from the `useBlockProps` react hook call.
```

Comment:
This phrasing feels a bit confusing to me but it could be a lack of technical knowledge. Specifically, this part: "but in order for the Gutenberg editor to know how to manipulate the block, add any extra classNames that are needed for the block...".

Reply:
I believe going into details might create even more confusion :P 
At this point in the learning curve of Gutenberg, people just need to know that this is important for Gutenberg to work well with their block. I'm not sure how I can express it better than that.